!movie add does not work

## Expected Behavior

To add movie to Radarr

## Actual Behavior

Errors:
    Error: Cannot find module './getRadarrSettings'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/opt/MediaButler/util/radarr/getQualityProfiles.js:1:78)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/opt/MediaButler/util/radarr/getQualityProfileId.js:1:90)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)

## Steps to Reproduce the Problem

  1. Start bot
  2. !movie add <movieId>

## Specifications

  - Version: v0.4
  - Platform: ALL

--- a/util/radarr/getQualityProfiles.js
+++ b/util/radarr/getQualityProfiles.js
@@ -1,5 +1,4 @@
-const getApi = require('./getRadarrSettings');
-const SonarrAPI = require('sonarr-api');
+const getApi = require('./getApi');
 module.exports = (guild) => {
   const p = new Promise((resolve, reject) =>
   {

